### DISCUSSION TITLE
Categories to Multiple Kitchen

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#400] [Ideas]
Hi Team,

We use a popular restaurant POS right now and used  many, but there is a mistake in the NexoPOS

The same categories cannot be applied to more than 1 kitchen. If a category is assigned to one kitchen then it should not be available on the other. And also it will be confusing for owners to find whether this category is assigned to the other kitchen or not if it keep showing available on all the kitchens.

**Comment by Blair2004** [comment]
Hi, strangely we had customers requesting to have some category shared among other kitchens. 
That's probably a feature to consider. Awaiting up voting for adding to the roadmap.

Thanks

**Reply by magedfathi** [reply]
some restaurants are printing the customer receipt and kitchen 1 and 2 products using the cashier printer and customers will receive their orders using this receipt

**Reply by Blair2004** [reply]
you can therefore assign the same printer for everything. Unless i didn't get your point.

**Reply by magedfathi** [reply]
yes, you are correct but in the main suggestion, he was asking that the category should be printed through only one printer which if you did will not support what you said so i thin the main suggestion is not good

**Reply by Blair2004** [reply]
Hi, 
i don't see why we should constraint to only have a category printted to one printer... We've made sure it's possible to either assign on a printer to one kitchen or assign multiple printer to the kitchen. If you want a kitchen to handle only one category and be printed to one printer, it's up to you to create a category that handle 1 category and is assigned to 1 printer.
